tests/dummy/config/targets.js

```diff
@@ -4,7 +4,7 @@ const browsers = [
   'last 1 Safari versions'
 ];
 
-const isCI = !!process.env.CI;
+const isCI = Boolean(process.env.CI);
```

Comment:
This came along with the default ruleset. I'd also be happy to set up an exception for `!!`, since it's one of the more easily readable type coersions.

Reply:
I think `Boolean(…)` is fine actually - might be a bit more readable for beginners 👍